README.md

```diff
@@ -57,6 +57,57 @@ You probably have enabled "Secure Window" feature in SealNote settings.
 Disable it to be able to take screenshots or show window content in application
 switcher.
 
+Contributing
+--------------------------------
+
+Contributions are much appreciated. Please go through this section to understand how and what to contribute.
+
+### Tools of trade
+
+1. Java 1.7+
+1. Gradle 0.12.2
```

Comment:
If I'm not mistaken, gradle version is not required as gradle script automatically downloads whatever dependencies except for Java and Android.